.github/workflows/updateIndex.yml

```diff
@@ -0,0 +1,53 @@
+
+name: Update Devices Index
+
+on:
+  pull_request:
+    types: [opened, edited]
+  issue_comment:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Setup repo
+        run: |
+          npm install
+          npm run prerelease
```

Comment:
That takes quite a bit of time. We could skip the entire job if no files in `packages/config/config/devices` are changed.

Reply:
How do I run the script without building ?

Reply:
No I mean looking at the git diff to see if there is even a need to run the script.

Reply:
The exit code of `git diff-index --name-only HEAD | grep packages/config/config/devices` should be 0 in that case

Reply:
Check it now